x-pack/plugins/cloud_security_posture/common/constants.ts

```diff
@@ -45,24 +45,23 @@ export const CLOUD_SECURITY_POSTURE_PACKAGE_NAME = 'cloud_security_posture';
 export const CDR_MISCONFIGURATIONS_DATA_VIEW_NAME = 'Latest Cloud Security Misconfigurations';
 export const CDR_MISCONFIGURATIONS_DATA_VIEW_ID_PREFIX =
   'security_solution_cdr_latest_misconfigurations';
-export const CDR_MISCONFIGURATIONS_INDEX_PATTERN =
-  'logs-*_latest_misconfigurations_cdr,logs-cloud_security_posture.findings_latest-default';
+export const CDR_LATEST_NATIVE_MISCONFIGURATIONS_INDEX_PATTERN =
+  'logs-cloud_security_posture.findings_latest-default';
+export const CDR_LATEST_THIRD_PARTY_MISCONFIGURATIONS_INDEX_PATTERN =
+  'logs-*_latest_misconfigurations_cdr';
```

Comment:
@maxcold, Wiz's index pattern `'logs-wiz.cloud_configuration_finding_latest-default'` doesn't match the CDR index pattern we agreed on, which is `'logs-*_latest_misconfigurations_cdr'`. Is this a temporary situation that will be resolved once the integration is ready?

Reply:
The data stream and transform haven't been implemented yet. If you are talking about my POC and draft PRs, than yes, they don't reflect our final proposal

Reply:
Thanks. So to make this PR compatible with the current Wiz integration POC, this row should be updated to:

```typescript
export const CDR_LATEST_THIRD_PARTY_MISCONFIGURATIONS_INDEX_PATTERN = 
      'logs-*_latest_misconfigurations_cdr,logs-wiz.cloud_configuration_finding_latest-default';
```

Reply:
Yep, that's correct, but we shouldn't add it to our codebase as it's very temporary